### ISSUE TITLE
Playground links from code blocks in books need to obey edition2018 flag

### ISSUE THREAD

**Issue by rchaser53** [rust-lang/rust#58491]
**EDIT**: The error here is actually from clicking the playground link, not that there's a problem with the code, or how the book's testpass is running.  Original description follows.

---

It seems that ```#![feature(try_blocks)]``` is not worked now.
link: https://doc.rust-lang.org/unstable-book/language-features/try-blocks.html

![build_failed](https://user-images.githubusercontent.com/9676954/52862079-9d36e300-3177-11e9-8ac5-413e00a54bba.png)

**Comment by Centril** [comment]
This is probably due to compiling the test with Rust 2015; it works on Rust 2018. cc @scottmcm

**Comment by scottmcm** [comment]
It definitely works in 2018: https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=e2ca081a42124a7615dd542c3cc36324  See also the run-pass test for it: https://github.com/rust-lang/rust/blob/master/src/test/run-pass/try-block.rs

It was originally marked ignore because of needing 2018 (https://github.com/rust-lang/rust/blob/master/src/test/run-pass/try-block.rs), but was changed to run in 2018 when https://github.com/rust-lang/rust/pull/54350 made that possible.

So I don't think there's a bug here.

**Comment by Centril** [comment]
@scottmcm The bug is in the unstable book linking the user towards a 2015 playground instead of a 2018 one.

**Comment by scottmcm** [comment]
@Centril Ok, renamed the issue and changed the flags.  This isn't a testsuite problem, then.

**Comment by Centril** [comment]
@scottmcm good call :) cc @steveklabnik

**Comment by scottmcm** [comment]
Related: https://github.com/rust-lang-nursery/mdBook/issues/812

**Comment by rconaway** [comment]
I need to use 2018 features in an MD book I'm creating.

It looks like this is a fresh issue, so I assume nobody has had a chance to deal with it yet.  In the meantime, is there any work-around I can use?  

I took a naive shot at changing a local copy of mdBook.  In src/theme/book.js, I changed the url that (I think) runs the code to 

`https://play.rust-lang.org/evaluate.json?version=stable&mode=debug&edition=2018`

That didn't seem to help any.   Any other suggestions?
